Lib/test/test_capi/test_number.py

```diff
@@ -0,0 +1,349 @@
+import itertools
+import operator
+import unittest
+import sys
+import warnings
+
+from test.support import cpython_only, import_helper
+from _testbuffer import ndarray
```

Comment:
Would it be possible to support missing _testbuffer extension? Only use ndarray if there is no ImportError.